src/box/lua/session.c

```diff
@@ -131,15 +131,15 @@ lbox_session_euid(struct lua_State *L)
 static int
 lbox_session_uid(struct lua_State *L)
 {
-	lua_pushnumber(L, current_session()->credentials.uid);
+	lua_pushnumber(L, current_session()->credentials.auth_uid);
 	return 1;
 }
 
 /** Session authenticated user name. */
 static int
 lbox_session_user(struct lua_State *L)
 {
-	struct user *user = user_by_id(current_session()->credentials.uid);
+	struct user *user = user_by_id(current_session()->credentials.auth_uid);
```

Comment:
The PR is split in a rather unfortunate way: introducing some struct member in one commit and using it in another complicates review. Better squash the two commits.